Explorer/Assets/DCL/Infrastructure/ECS/SceneLifeCycle/Systems/EarlyAsset/EarlySceneRequestSystem.cs

```diff
@@ -0,0 +1,86 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using DCL.Ipfs;
+using DCL.RealmNavigation;
+using DCL.Utility;
+using DefaultNamespace;
+using ECS.Abstract;
+using ECS.Prioritization.Components;
+using ECS.SceneLifeCycle.SceneDefinition;
+using ECS.StreamableLoading.AssetBundles;
+using ECS.StreamableLoading.Common;
+using ECS.StreamableLoading.Common.Components;
+using System.Collections.Generic;
+using Unity.Mathematics;
+using UnityEngine;
+using Utility;
+using ScenePromise = ECS.StreamableLoading.Common.AssetPromise<DCL.Ipfs.SceneDefinitions, ECS.SceneLifeCycle.SceneDefinition.GetSceneDefinitionList>;
+
+
+namespace ECS.SceneLifeCycle.Systems.EarlyAsset
+{
+    [UpdateInGroup(typeof(PresentationSystemGroup))]
+    [UpdateBefore(typeof(EarlyAssetBundleRequestSystem))]
+    public partial class EarlySceneRequestSystem : BaseUnityLoopSystem
```

Comment:
I am not quite sure what problem you resolve with this system, but it looks like the same logic should work on teleportation as well